samples/msal-node-samples/on-behalf-of/index.js

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
we probably need to update the README as well if we'll use a single project instead of 2.

However I would much prefer maintaining the separate web app and web api projects if possible (at least separate .js files for each under the same project).

Reply:
I didn't touch the original folders for the web app and web api in here, so the readme is still accurate. Perhaps I could add info to the readme that further explains how to run the sample via index.js. Thoughts?

Reply:
Or we could exclude it - however the functionality inside index.js is needed for the jest tests.

Reply:
OK, in that case can we move index.js (and others related to the test) under the test/ folder then?